setup.cfg

```diff
@@ -6,6 +6,7 @@ testpaths = test
 # Tests exist for BootstrapBuilder, which runs a debootstrap in a temporary directory. debootstrap cannot
 # run in /tmp because it's usually mounted with nodev,nosuid options, hence we use /var/tmp here.
 addopts = --basetemp=/var/tmp/furnace-pytest
+filterwarnings = error::DeprecationWarning
```

Comment:
I think we should do this the same we did in the I think, we should do this as the same we did in the https://github.com/OneIdentity/bldr/pull/16/files.

Reply:
We don't have problems with distutils' old Version class here, because furnace does not require the docker package.

Reply:
Ohh, alright. Thank you!